live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -76,3 +76,8 @@
   delay: 30
   register: result
   until: result is succeeded
+
+#
+# CRA on Linux requires that the engine UUID is stored in a file
+#
+- shell: dmidecode -s system-uuid | awk '{print tolower($0)}' > /etc/engine-uuid
```

Comment:
I assume this requirement exists so that a given set of private keying material remains valid and verifiable even if the underlying `system-uuid` changes?

Are there any valid customer use cases that result in a change in `system-uuid`? For example, some customers have been known to create multiple engines by cloning a pre-setup/pre-registration image. If the `/etc/engine-uuid` file is identical in all of those clones, then I won't that be problematic?

Reply:
I'm undoing this part of the change and replacing it with http://reviews.delphix.com/r/50879/ which gets rid of the dependency of CRA on that file altogether.